src/redux/books/books.js

```diff
@@ -1,14 +1,63 @@
-import React from 'react';
-
-const BooksPage = () => (
-  <>
-    <div className="books-container d-flex">
-      <h2>Book List</h2>
-      <button type="button">Remove</button>
-    </div>
-    <input type="text" name="addBooks" placeholder="Book Title" />
-    <button type="button">Add Book</button>
-  </>
-);
-
-export default BooksPage;
+import { v4 as uuidv4 } from 'uuid';
+
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+// const initialState = [];
+const initialState = [
+  {
+    id: uuidv4(),
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+    genre: 'Action',
+    completed: 64,
+  },
+  {
+    id: uuidv4(),
+    title: 'Dune',
+    author: 'Frank Herbert',
+    genre: 'Science Fiction',
+    completed: 8,
+  },
+  {
+    id: uuidv4(),
+    title: 'Capital in the Twenty-First Century',
+    author: 'Suzanne Collins',
+    genre: 'Economy',
+    completed: 0,
+  },
+];
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      /* return [ ...state, action.payload]; */
```

Comment:
- [ ] Please delete the unnecessary commented code, this way your code will look cleaner 😄
    - Take a look at your project and if you have more commented code like this, please delete it 🚀